contrib/experimental/great_expectations_experimental/metrics/column_values_string_integers_monotonically_increasing.py

```diff
@@ -0,0 +1,153 @@
+    function_metric_name = "column_values.string_integers.monotonically_increasing"
+
+    @column_function_partial(engine=PandasExecutionEngine)
+    def _pandas(self, data, _metrics, **kwargs):
+
+        try:
+            temp_column = [int(x) for x in data if x.isdigit()]
+        except AttributeError:
+            raise TypeError(
+                "Column must be a string-type capable of being cast to int."
+            )
+
+        if len(temp_column) != len(data):
+            raise TypeError(
+                "Column must be a string-type capable of being cast to int."
+            )
+
+        series_diff = np.diff(temp_column)
+
+        return series_diff >= 0
+
+    @metric_partial(
+        engine=SparkDFExecutionEngine,
+        partial_fn_type=MetricPartialFunctionTypes.WINDOW_FN,
+        domain_type=MetricDomainTypes.COLUMN,
+    )
+    def _spark(
+        cls,
+        execution_engine,
+        metric_domain_kwargs,
+        metric_value_kwargs,
+        metrics,
+        runtime_configuration,
+    ):
+        column_name = metric_domain_kwargs["column"]
+        table_columns = metrics["table.column_types"]
+        column_metadata = [col for col in table_columns if col["name"] == column_name][
+            0
+        ]
+
+        if isinstance(column_metadata["type"], (sparktypes.StringType)):
+            try:
+                column = F.col(column_name).cast(sparktypes.IntegerType())
+            except TypeError:
+                raise TypeError(
+                    "Column must be a string-type capable of being cast to int."
+                )
+        else:
+            raise TypeError(
+                "Column must be a string-type capable of being cast to int."
+            )
+
+        compute_domain_kwargs = metric_domain_kwargs
+
+        (
+            df,
+            compute_domain_kwargs,
+            accessor_domain_kwargs,
+        ) = execution_engine.get_compute_domain(
+            compute_domain_kwargs, domain_type=MetricDomainTypes.COLUMN
+        )
+
+        if any(np.array(df.select(column.isNull()).collect())):
+            raise TypeError(
+                "Column must be a string-type capable of being cast to int."
+            )
+
+        diff = column - F.lag(column).over(Window.orderBy(F.lit("constant")))
+        diff = F.when(diff.isNull(), 1).otherwise(diff)
+        diff = F.when(diff < 0, F.lit(False)).otherwise(F.lit(True))
+
+        return (
+            np.array(df.select(diff).collect()).reshape(-1)[1:],
+            compute_domain_kwargs,
+            accessor_domain_kwargs,
+        )
+
+    @metric_partial(
+        engine=SqlAlchemyExecutionEngine,
+        partial_fn_type=MetricPartialFunctionTypes.WINDOW_FN,
+        domain_type=MetricDomainTypes.COLUMN,
+    )
+    def _sqlalchemy(
+        cls,
+        execution_engine,
+        metric_domain_kwargs,
+        metric_value_kwargs,
+        metrics,
+        runtime_configuration,
+    ):
+        pass
```

Comment:
Based on what we see in `column_values_increasing.py` this may not be possible and can be removed.

Reply:
@alexsherstinsky Is this doable / desirable? Love to get your voice on if (or how) to approach this.

Reply:
@austiezr I am undecided regarding whether or not this expectation is desirable.  I am leaning more toward the negative with the justification that converting the column of strings to a column of integers could be a pre-processing steps that users could do before applying our existing expectation.  Perhaps it could be part of a category in our user gallery of expectations.  I may be missing a point, so happy to discuss.  Thank you very much!

Reply:
Not implementing, as discussed.